daemon/src/rollover_maker.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is this? :)

Reply:
Classic! I'll see how many files were not moved properly.